### DISCUSSION TITLE
Validation Error - PDF Portfolio/Attachment Extraction

### DISCUSSION THREAD

**Discussion by inkwell84** [pdfcpu/pdfcpu#658] [Q&A]
Hello, 

I am not sure this is a Bug so I did not want to submit it as an "Issue." Perhaps with your guidance we may determine that it is?

I am trying to extract PDF files that attached to a PDF Portfolio file.

I run `pdfcpu validate filename.pdf ` and I get this error:

```
validating(mode=relaxed) Portfolio of order #8_6-16-23.pdf ...
validation error (obj#:127): pdfcpu: validateNameEntry: dict=colFlddict entry=Subtype invalid dict entry: AFRelationship
```

I run `pdfcpu portfolio list filename.pdf` and I get this error:

```
pdfcpu: validateNameEntry: dict=colFlddict entry=Subtype invalid dict entry: AFRelationship
```

I run `pdfcpu attachment list filename.pdf` and I get this error:

```
pdfcpu: validateNameEntry: dict=colFlddict entry=Subtype invalid dict entry: AFRelationship
```

Can anyone help me understand what's going on (what can I do to fix?)

**Comment by hhrutter** [comment]
Hi there,

**AFRelationship** is usually associated with File Specification dictionaries.
It definitely is not an allowed value for the subtype of a Collection Field dictionary and therefore does not pass validation.
Who is the producer / PDF writer of your file?
If you can share the file create an issue and I will look into this.

**Reply by inkwell84** [reply]
Can I email/dm this to you instead of uploading?

**Reply by hhrutter** [reply]
hhrutter@gmail

**Reply by inkwell84** [reply]
Sent! Do you want me to create an "issue"? Or just continue to dialogue in this "discussion"?

Thank you!

**Comment by hhrutter** [comment]
see #659
